views/components/Game.jsx

```diff
@@ -1,25 +1,24 @@
 import React from 'react';
-import GameScene from './GameScene';
-import Map from './Map';
-import Character from './Character';
-import * as BrowserActions from '../actions/Browser';
-
-class Game extends React.Component {
-  componentWillMount() {
-    this.resizeGameScene();
-  }
+import GameScene from '../components/GameScene';
+import Map from '../components/Map';
+import Tree from '../components/Tree';
+import Character from '../containers/Character';
+import Exit from '../components/Exit'
+import keydrown from 'keydrown';
+const ReactTHREE = require('react-three');
```

Comment:
Separate imports from requires with one line, and import relative paths secondly.